tests/integration/targets/rds_global_cluster_create/tasks/main.yaml

```diff
@@ -0,0 +1,109 @@
+- module_defaults:
+    group/aws:
+      region: "{{ aws_region }}"
+      access_key: "{{ aws_access_key }}"
+      secret_key: "{{ aws_secret_key }}"
+      session_token: "{{ security_token | default(omit) }}"
+    group/amazon.cloud.aws:
+      aws_access_key: "{{ aws_access_key }}"
+      aws_secret_key: "{{ aws_secret_key }}"
+      security_token: "{{ security_token | default(omit) }}"
+      region: "{{ aws_region }}"
+  block:
+    - name: Create aurora global cluster
+      amazon.cloud.rds_global_cluster:
+        global_cluster_identifier: "{{ global_cluster_id }}"
+        engine: "{{ engine }}"
+        engine_version: "{{ engine_version }}"
+        region: "{{ aws_region }}"
+        state: present
+
+    - name: Create a primary cluster for global database
+      amazon.aws.rds_cluster:
+        db_cluster_identifier: "{{ primary_cluster_id }}"
+        engine: "{{ engine }}"
+        engine_version: "{{ engine_version }}"
+        master_username: "{{ username }}"
+        master_user_password: "{{ password }}"
+        global_cluster_identifier: "{{ global_cluster_id }}"
+        region: "{{ aws_region }}"
+      register: primary_cluster
+
+    - name: Create an instance connected to primary cluster
+      amazon.aws.rds_instance:
+        db_cluster_identifier: "{{ primary_cluster_id }}"
+        db_instance_identifier: "{{ primary_instance_id }}"
+        region: "{{ aws_region }}"
+        engine: "{{ engine }}"
+        db_instance_class: "{{ instance_class }}"
+
+    - name: Create a read replica cluster for global database
+      amazon.aws.rds_cluster:
+        db_cluster_identifier: "{{ secondary_cluster_id }}"
+        region: "eu-north-1"
+        engine: "{{ engine }}"
+        engine_version: "{{ engine_version }}"
+        global_cluster_identifier: "{{ global_cluster_id }}"
+      register: replica_cluster
+
+    - name: Get Global DB information
```

Comment:
Can you also please another task using filters rather than  global_cluster_identifier and some asserts?